pkgs/tools/typesetting/tex/latexrun/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenvNoCC, fetchFromGitHub, python3 }:
+
+stdenvNoCC.mkDerivation {
```

Comment:
I wonder if this would still need to use `buildPythonApplication`. cc @FRidh

Reply:
I tried that first, but it complained about needing a `setup.py`. Since this package doesn't require any libraries other than the standard library, I figured this is fine.